src/sentry/snuba/metrics/fields/base.py

```diff
@@ -431,45 +573,9 @@ def __recursively_generate_singular_entity_constituents(cls, projects, derived_m
         return entities_and_metric_names
 
     def generate_bottom_up_derived_metrics_dependencies(self):
-        """
-        Function that builds a metrics dependency list from a derived metric_tree
-        As an example, let's consider the `session.errored` derived metric
-
-            session.errored
-               /   \
-              /    session.errored_preaggregated
-             /
-        session.errored_set
-
-        This function would generate a bottom up dependency list that would look something like
-        this ["session.errored_set", "session.errored_preaggregated", "session.errored"]
-
-        This is necessary especially for instances of `CompositeEntityDerivedMetric` because these
-        do not have a direct mapping to a query alias but are rather computed post query, and so to
-        calculate the value of that field we would need to guarantee that the values of its
-        constituent metrics are computed first.
-
-        This is more apparent when the dependency tree contains multiple levels of instances of
-        `CompositeEntityDerivedMetric`. Following up with our example,
-
-        session.errored
-               /   \
-              /    composite_entity_derived_metric
-             /       \
-            /        session.errored_preaggregated
-           /
-        session.errored_set
-
-        In this modified example, our dependency list would change to
-        [
-        "session.errored_set", "session.errored_preaggregated",
-        "composite_entity_derived_metric", "session.errored"
-        ]
-        And this order is necessary because we would loop over this list to compute
-        `composite_entity_derived_metric` first which does not have a direct mapping to a query
-        alias before we are able to compute `session.errored` which also does not have a direct
-        query alias
-        """
+        # We are only interested in the dependency tree from instances of
```

Comment:
I moved some doc blocks around because this method now also lives in the baseclass